frontend/sass/components/_modal.scss

```diff
@@ -0,0 +1,61 @@
+@use "../abstracts" as abstracts;
+
+.modal-background {
+  display: flex;
+  position: fixed;
+  background-color: rgba(abstracts.$primary-dark, 0.5);
+  justify-content: center;
+  align-items: center;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+}
+
+.modal-box {
+  background-color: abstracts.$primary-dark;
+  display: flex;
+  flex-direction: column;
+  color: white;
+  width: 700px;
+  height: 400px;
+}
+
+.modal-title {
+  display: flex;
+  justify-content: flex-end;
+  margin-top: 5px;
```

Comment:
Marginit voisi muuttaa paddingiksi, ettei modal-boxin taustaväri näy reunoilta. Halutessaan marginit/paddingit voi määritellä myös yhdellä rivillä näin: padding: 5px 10px. Ensimmäinen numero vaikuttaa ylä- ja alareunoihin, toinen sivuille.